src/tabular-api-surface.ts

```diff
@@ -674,13 +681,21 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
         return pendingEntryIndices.has(index);
       });
 
-      const reqOpts = {
-        tableName: this.name,
+      const baseReqOpts = (
+        this.viewName
+          ? {
+              authorizedViewName: `${this.name}/authorizedViews/${this.viewName}`,
+            }
+          : {
+              tableName: this.name,
+            }
+      ) as google.bigtable.v2.IReadRowsRequest;
+      const reqOpts = Object.assign(baseReqOpts, {
```

Comment:
This change makes it so that when viewName is defined, ie. when the object is an authorized view that the request will be made for an authorized view and not a table.

Reply:
Same as above: maybe explain this with a comment?

Reply:
Done